openwisp_radius/tasks.py

```diff
@@ -147,15 +148,22 @@ def get_radsecret_from_radacct(rad_acct):
                 )
 
     def get_radius_reply_name_and_value(user, check):
-        Counter = app_settings.CHECK_ATTRIBUTE_COUNTERS_MAP[check.attribute]
-        counter = Counter(user=user, group=check.group, group_check=check)
         try:
+            Counter = app_settings.CHECK_ATTRIBUTE_COUNTERS_MAP[check.attribute]
+            counter = Counter(user=user, group=check.group, group_check=check)
             value = counter.check()
             return counter.reply_name, value
-        except KeyError:
-            return check.attribute, check.value
         except Exception as e:
-            logger.exception(f"Got {e} while CoA for counter {Counter}")
+            # Ignore MaxQuotaReached and KeyError exceptions:
+            # - MaxQuotaReached: raised by Counter.check() when user has
```

Comment:
If `MaxQuotaReached` is raised, wouldn't be more correct to `return  counter.reply_name, 0`?

Or am I missing something?